nginx/nginx.go

```diff
@@ -95,7 +95,10 @@ func (n *nginxUpdater) signalIfRequired() {
 	if n.updateRequired.Get() {
 		log.Info("Signalling Nginx to reload configuration")
 		incrementReloadMetric()
```

Comment:
can we move this after we have sent the signal pls

Reply:
*both lines

Reply:
done